source/_integrations/tomorrowio.markdown

```diff
@@ -0,0 +1,36 @@
+---
+title: Tomorrow.io
+description: Instructions on how to integrate the Tomorrow.io Weather and Air Quality API into Home Assistant.
+ha_category:
+  - Environment
+  - Health
+  - Sensor
+  - Weather
+ha_release: 2021.9
+ha_iot_class: Cloud Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@raman325'
+ha_domain: tomorrowio
+ha_platforms:
+  - sensor
+  - weather
+---
+
+The Tomorrow.io integration allows you to obtain weather, air quality, pollen, and fire information from the [Tomorrow.io API](https://www.tomorrow.io/weather-api/).
+
+## Obtain an API key
+
+You can obtain a free API key by signing up with [Tomorrow.io](https://www.tomorrow.io/weather-api/).
+
+The integration will automatically set the refresh interval based on the number of Tomorrow.io integrations that are using the same API key for a given Home Assistant instance. The integration currently assumes you are using a free account so the max requests per day is 500 (the refresh interval is calculated such that you should only use up around 90% of the quota). If you want to use a paid account to increase the number of max requests and increase the update frequency, please open a Home Assistant issue for the integration.
```

Comment:
> If you want to use a paid account to increase the number of max requests and increase the update frequency, please open a Home Assistant issue for the integration.

? So it is possible? It isn't possible? Its a bug and you want people to report it? 😬

Reply:
lol fair enough, I will remove that